docstr_coverage/printers.py

```diff
@@ -22,128 +24,440 @@
+        * `verbosity` equal `1` - prints all overall coverage statistic.
+        * `verbosity` equal `2` - prints all overall coverage statistic and *needed*, *found*,
+          *missing* and *coverage* file statistics.
+        * `verbosity` equal `3` - prints all overall coverage statistic, information about *nodes
+          without docstrings*, *needed*, *found*, *missing* and *coverage* file statistics.
+        * `verbosity` equal `4` - prints all overall coverage statistic, information about *empty
+          files*, *nodes with docstrings*, *ignored nodes*, *nodes without docstrings*, *needed*,
+          *found*, *missing* and *coverage* file statistics.
+        """
+        pass
 
-            # List of missing docstrings
-            if self.verbosity >= 3:
-                if file.status == FileStatus.EMPTY and self.verbosity > 3:
-                    print_line(" - File is empty")
-                for expected_docstr in file._expected_docstrings:
-                    if expected_docstr.has_docstring and self.verbosity > 3:
-                        print_line(
-                            " - Found docstring for `{0}`".format(expected_docstr.node_identifier)
-                        )
-                    elif expected_docstr.ignore_reason and self.verbosity > 3:
-                        print_line(
-                            " - Ignored `{0}`: reason: `{1}`".format(
-                                expected_docstr.node_identifier, expected_docstr.ignore_reason
-                            )
-                        )
-                    elif not expected_docstr.has_docstring and not expected_docstr.ignore_reason:
-                        if expected_docstr.node_identifier == "module docstring":
-                            print_line(" - No module docstring")
-                        else:
-                            print_line(
-                                " - No docstring for `{0}`".format(expected_docstr.node_identifier)
-                            )
-
-            # Statistics
-            count = file.count_aggregate()
-            print_line(
-                " Needed: %s; Found: %s; Missing: %s; Coverage: %.1f%%"
-                % (
-                    count.needed,
-                    count.found,
-                    count.missing,
-                    count.coverage(),
-                ),
-            )
-            print_line()
-        print_line()
 
-    def _print_overall_statistics(self, results):
-        """Prints overall results (aggregated over all files) to stdout.
+class LegacyPrinter(Printer):
+    """Print in simple format to output"""
+
+    def _print_line(self, line: str = ""):
+        """Prints `line`
 
         Parameters
         ----------
-        results: ResultCollection
-            The information about docstr presence to be printed to stdout."""
-        count = results.count_aggregate()
+        line: String
+            The text to print"""
+        logger.info(line)
```

Comment:
Is there a reason you wanted to move this to be a method of `LegacyPrinter`? I preferred it as a top-level helper function since it doesn't need to be connected to any instance anyway. 

I was questioning whether we even need this anymore since it just calls `logger.info`. However, maybe we need to close out the discussion on `logging` vs `print` first.

Reply:
@skv0zsneg I believe this one is not yet addressed?

Reply:
@MiWeiss @HunterMcGushion 
In last updates I moved this print line feature (now renamed to `print_to_stdout`) to LegacyPrinter only. MarkDown is inherited from LegacyPrinter and uses his print.

I decided to made it this way because makrdown is a special case of simple print (LegacyPrinter).